client/src/component/Header/Header.jsx

```diff
@@ -1,11 +1,79 @@
-import React from 'react'
+import React, {useState} from 'react'
+import {NavLink} from "react-router-dom";
+import {AppBar, Avatar, Toolbar, IconButton, Menu, Typography, MenuItem, Button} from '@material-ui/core'
 
 
+import useStyles from './style'
+
 function Header() {
+    const classes = useStyles();
+
+   const [auth, setAuth] = useState(false);
+   const [anchorEl, setAnchorEl] = useState(null);
+   const open = Boolean(anchorEl);
+
+  
+  const handleMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  
+
     return (
-        <div>
-            
-        </div>
+        <div className={classes.root}>
+        <AppBar position="static" className={classes.appbar}>
+          <Toolbar>
+            <IconButton edge="start" className={classes.menuButton} color="inherit" aria-label="menu">
+            </IconButton>
+            <Typography variant="h6" className={classes.title}>
+              Travel Booking
+            </Typography>
+            <div className={classes.pages}>
+                    <NavLink className={classes.navlinks} to="#">Explore</NavLink>
+                    <NavLink className={classes.navlinks} to="/">Flights</NavLink>
+                    <NavLink className={classes.navlinks} to="/hotel" activeStyle={{ color: "#FFA000"}}>Hotels</NavLink>
+                    <NavLink className={classes.navlinks} to="/rent" activeStyle={{ color: "#FFA000"}}>Rent</NavLink>
+             </div>
+          
+            {auth ? (
+              <div>
+                <IconButton
+                  aria-label="account of current user"
+                  aria-controls="menu-appbar"
+                  aria-haspopup="true"
+                  onClick={handleMenu}
+                  color="inherit"
+                >
+                  <Avatar className={classes.avatar} src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTdX7wWCMOvGYD6_4-MthVKf-DjjgLF_GqQzg&usqp=CAU" />
+
+                </IconButton>
+                <Menu
+                  id="menu-appbar"
+                  anchorEl={anchorEl}
+                  anchorOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                  }}
+                  keepMounted
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'right',
+                  }}
+                  open={open}
+                  onClose={handleClose}
+                >
+                  <MenuItem onClick={handleClose}>Profile</MenuItem>
+                  <MenuItem onClick={handleClose}>Logout</MenuItem>
+                </Menu>
+              </div>
+            ) : <Button onClick={()=> setAuth(true)} className={classes.loginbtn}>Login</Button>}
```

Comment:
Works for now. However, as the logic to authenticate becomes more complex, you might want to refactor this to make it easier to read/understand.